Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -399,6 +402,12 @@ def plus_char(a, b):
 def minus_char(a, b):
     return chr(ord(a) - ord(b))
 
+def plus_unicode_char(a, b):
+    return CodePoint(plus_char(a, b))
```

Comment:
I was sure to have made this remark elsewhere, but can't find it, so: Can you overload `+` and `-` instead?

Reply:
Ah true, I can do that. For the record I don't care much about the usability of `CodePoint` though, since I only expect compiled Dafny code to reference it.

Reply:
It did end up simplifying the code generation logic a little though, which I DO care about :) , so thanks!